test/fn/edit-contacts.spec.js

```diff
@@ -0,0 +1,197 @@
+const { expect, assert } = require('chai');
+const rewire = require('rewire');
+const PouchDB = require('pouchdb-core');
+PouchDB.plugin(require('pouchdb-adapter-memory'));
+const fs = require('../../src/lib/sync-fs');
+const environment = require('../../src/lib/environment');
+const sinon = require('sinon');
+// const info = require('../../src/lib/log');
+
+let pouch, editContactsModule;
+
+// specifying directory paths to use
+const editContactsPath = `data/edit-contacts`;
+const saveDocsDir = `${editContactsPath}/json_docs`;
+const expectedDocsDirNested = `${editContactsPath}/expected-json_docs/nested-columns`;
+const expectedDocsDirOneCol = `${editContactsPath}/expected-json_docs/one-column`;
+const expectedDocsDirAllCols = `${editContactsPath}/expected-json_docs/all-columns`;
+const expectedDocsMultipleCsv = `${editContactsPath}/expected-json_docs/multiple-csv-columns`;
+const expectedDocsUsersCsv = `${editContactsPath}/expected-json_docs/user-columns`;
+const filesToUpload = fs.recurseFiles(`${editContactsPath}/server-contact_docs`).filter(name => name.endsWith('.json'));
+const countFilesInDir = path => fs.fs.readdirSync(path).length;
+
+const docs = filesToUpload
+  .map(file => {
+    const doc = fs.readJson(file);
+    return doc;
+  });
+
+const uploadDocuments = (docs) => {
+  return pouch.bulkDocs(docs);
+};
+
+function compareDocuments(expectedDocsDir){
```

Comment:
consider changing this to use `const <function_name> = ....`, for consistency :)